src/plugin/kubearmor.go

```diff
@@ -49,14 +50,20 @@ func ConvertKnoxSystemPolicyToKubeArmorPolicy(knoxPolicies []types.KnoxSystemPol
 		kubePolicy := types.KubeArmorPolicy{
 			APIVersion: "security.kubearmor.com/v1",
 			Kind:       "KubeArmorPolicy",
-			Metadata:   map[string]string{},
+			Metadata: metav1.ObjectMeta{
+				Name:        "",
+				Namespace:   "",
+				Annotations: map[string]string{},
```

Comment:
As we are following `Setting Fields After Declaration` approach.

```
kubePolicy := types.KubeArmorPolicy{
	APIVersion: "security.kubearmor.com/v1",
	Kind:       "KubeArmorPolicy",
}

// Set fields after declaration
kubePolicy.Metadata.Name = xxxxxxxxx
kubePolicy.Metadata.Annotations = map[string]string{
	"tldr":        "zzzzzz",
	"description": "yyyyyy",
}
```
By not providing an empty ObjectMeta{} during declaration, we avoid unnecessary initialization of the Metadata field, which makes the code slightly more efficient. When we set the fields later, Go will automatically initialize Metadata as an empty ObjectMeta{} when needed.

Reply:
addressed